### PR TITLE
Accelerator interface

### PR DESCRIPTION
**Pull request by yo96**
FL, CL, and RTL accelerator interface implementaions.

### CODE REVIEW

**Comment by jsn1993** [comment]
Are you going to finish RTL/CL adapters in this PR?

**Comment by yo96** [comment]
There is no need for RTL/CL adapters for xcel interface. The default connect by name just works. See my test for an example. I'm not sure whether I should implement FL/RTL adapters though.

**Comment by jsn1993** [comment]
> There is no need for RTL/CL adapters for xcel interface. The default connect by name just works. See my test for an example. I'm not sure whether I should implement FL/RTL adapters though.

hmmm you can implement it in FL I guess

File: pymtl3/stdlib/ifcs/xcel_ifcs.py [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+        s.left.resp.en  = Bits1(1)
+        s.left.resp.msg = resp
+    
+    @s.update
+    def up_xcelifc_rtl_fl_rdy():
+      s.left.req.rdy = s.left.resp.rdy
+
+# FIXME: This adapter doesn't work.
+class XcelIfcFL2RTLAdapter( Component ):
+
```
Comment (yo96) [inline pymtl3/stdlib/ifcs/xcel_ifcs.py]: @jsn1993 I can't get this to work.

File: pymtl3/stdlib/ifcs/xcel_ifcs.py [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+      parent.connect_pairs(
+        s,       m.left,
+        m.right, other,
+      )
+      parent.XcelIfcFL2RTL_count += 1
+      return True
+
+    # TODO: figure out clean way to do this or unify all special connect
+    #       in FL.
+    return other.connect( s, parent )
```
Comment (yo96) [inline pymtl3/stdlib/ifcs/xcel_ifcs.py]: @jsn1993 I have to do this otherwise it will not connect to CL. Thoughts?

**Pull request merged**